### PR TITLE
[14.0][FIX] account_loan: account.loan.pay.amount, crud permission to group_account_manager

### PR DESCRIPTION
**Pull request by GSLabIt**
Allow manager to create, write, delete: `account.loan.pay.amount`.  Actually user has read only permission and it raise an error. See video:

https://github.com/OCA/account-financial-tools/assets/11652845/86e523b4-aaf7-44be-8f54-0c586a8c394c

### CODE REVIEW

**Comment by francesco-ooops** [comment]
@etobella can I ask you to take a look at this? Thanks!

File: account_loan/security/ir.model.access.csv [hunk 1]
```diff
@@ -7,3 +7,4 @@ access_account_loan_generate_wizard,access_account_loan_generate_wizard,model_ac
 access_account_loan_pay_amount,access_account_loan_pay_amount,model_account_loan_pay_amount,account.group_account_user,1,0,0,0
```
Comment (etobella) [inline account_loan/security/ir.model.access.csv]: wouldn't be better to set here 1,1,1,0 and not adding the new line?

Comment (GSLabIt) [reply account_loan/security/ir.model.access.csv]: what about if manager needs to delete for any reason?

Comment (etobella) [reply account_loan/security/ir.model.access.csv]: it's a wizard, there is no need to delete :wink:

Comment (GSLabIt) [reply account_loan/security/ir.model.access.csv]: ah ok

**Review by etobella — APPROVED** [review]

**Review by PicchiSeba — APPROVED** [review]
Code review: LGTM

**Comment by francesco-ooops** [comment]
@etobella shall we go? we can fw-port it

**Pull request merged**